### PR TITLE
Navbar SCSS

### PR DESCRIPTION
**Pull request by cchanyu**

### CODE REVIEW

File: src/components/navbar/navbar.js [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
     render() {
         return (
-            <Router className = "route">
-                <div>
-                    <ul>
-                        <li>
-                            <NavLink activeStyle={{ background: "lightgray" }} to="/home">Home</NavLink>
+            <Router className="navbar">
+                <div className="navbar__container">
+                    <ul className="navbar__unorderedlist">
```
Comment (richsong) [inline src/components/navbar/navbar.js]: Let's move away from naming things as unordered list. What is this unorderlist? Is it a Navbar item? Navbar link?

Comment (cchanyu) [reply src/components/navbar/navbar.js]: Okay, I renamed to Navbar listcontainer.

File: src/components/navbar/navbar.js [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
+                            <NavLink className="navbar__link" to="/home">Home</NavLink>
                         </li>
-                        <Link to="/home">
-                            <div>
-                                {/* {<img src={Sakslogo} />} */}
-                                <h2>List View</h2>
+                        <Link to="/">
+                            <div className="navbar__home">
+                                <div className="navbar__logo"></div>
+                                <div className="navbar__title">HBC Digital</div>
```
Comment (richsong) [inline src/components/navbar/navbar.js]: Please think about restructuring this like so:
```
Navbar
  Navbarmenu
    Navbar title?
    NavbarLogo?
    Navbar links/button/categories/pane
```

File: src/components/navbar/navbar.js [hunk 3]
```diff
@@ -1,27 +1,26 @@
...
-                        <Link to="/home">
-                            <div>
-                                {/* {<img src={Sakslogo} />} */}
-                                <h2>List View</h2>
+            <Router className="navbar">
+                <div className="navbar__container">
+                    <ul className="navbar__menulist">
+                        <Link to="/">
+                            <div className="navbar__linktoroot">
+                                <div className="navbar__hbctitle">HBC Digital</div>
```
Comment (richsong) [inline src/components/navbar/navbar.js]: rename to this format: `navbar__title` or `navbar__hbc-title`

File: src/components/navbar/navbar.js [hunk 4]
```diff
@@ -1,27 +1,26 @@
...
+                                <div className="navbar__hbclogo"></div>
```
Comment (richsong) [inline src/components/navbar/navbar.js]: Same here

File: src/components/navbar/navbar.js [hunk 5]
```diff
@@ -1,27 +1,26 @@
...
-                        </li>
-                        <Link to="/home">
-                            <div>
-                                {/* {<img src={Sakslogo} />} */}
-                                <h2>List View</h2>
+            <Router className="navbar">
+                <div className="navbar__container">
+                    <ul className="navbar__menulist">
+                        <Link to="/">
+                            <div className="navbar__linktoroot">
```
Comment (richsong) [inline src/components/navbar/navbar.js]: Maybe just navbar__home

File: src/components/navbar/navbar.js [hunk 6]
```diff
@@ -1,27 +1,26 @@
...
+                <div className="navbar__container">
+                    <ul className="navbar__menulist">
+                        <Link to="/">
+                            <div className="navbar__home">
+                                <div className="navbar__hbc-title">HBC Digital</div>
+                                <div className="navbar__hbc-logo"></div>
                             </div>
                         </Link>
+                        <li className="navbar__linklist">
+                            <NavLink className="navbar__linktohome" to="/home">Home</NavLink>
```
Comment (richsong) [inline src/components/navbar/navbar.js]: `navbar__linktohome` - Will this particular link be unique to only the home button? Do you plan on creating more links in the future? If you do, do you please on reusing this class? Perhaps we should rename this to `navbar__link`. I can see a potential case where you will have a few links in the Navbar... all of which should have the same class. Unless there is some special styling for a special link.

File: src/components/navbar/navbar.js [hunk 7]
```diff
@@ -1,27 +1,26 @@
...
+            <Router className="navbar">
+                <div className="navbar__container">
+                    <ul className="navbar__menulist">
+                        <Link to="/">
+                            <div className="navbar__home">
+                                <div className="navbar__hbc-title">HBC Digital</div>
+                                <div className="navbar__hbc-logo"></div>
                             </div>
                         </Link>
+                        <li className="navbar__linklist">
```
Comment (richsong) [inline src/components/navbar/navbar.js]: Does this look right to you? Take a look at line 17 and 18

File: src/components/navbar/navbar.js [hunk 8]
```diff
@@ -1,27 +1,26 @@
 import React, { Component } from 'react';
 import { Route, NavLink, Link, Switch, BrowserRouter as Router } from 'react-router-dom';
 
-import './navbar.css'
-import './navbar.scss'
-import Home from '../Home/home';
+import './Navbar.scss'
+import Home from '../Home/Home';
```
Comment (richsong) [inline src/components/navbar/navbar.js]: What is this? How do we plan on using it? Where is it here?

**Pull request merged**